build-environment.yml

```
@@ -5,5 +5,5 @@ dependencies:
   - python
   - pip
   - pip:
-    - jupyterlite-xeus>=0.1.5,<0.2
-    - voici>=0.5,<0.6
+    - jupyterlite-xeus>=0.1.8,<0.2
+    - voici-core>=0.6,<0.7
```

Actually, it could be the following?

```suggestion
    - voici=0.6,<0.7
```

---

Yes. Wondering if leaving the two `voici-core` and `jupyterlite-xeus` would make it easier to use the Pyodide kernel instead if folks want to.

---

So they could replace the `jupyterlite-xeus` by `jupyterlite-pyodide-kernel`.

---

Sounds like a good idea yes 👍🏽 let's keep it that way then